cns/restserver/api.go

```diff
@@ -408,6 +408,12 @@ func (service *HTTPRestService) reserveIPAddress(w http.ResponseWriter, r *http.
 		Message:    returnMessage,
 	}
 
+	if resp.ReturnCode == 0 {
+		// If Response is success i.e. code 0, then publish metrics.
+		state := service.buildIPState()
+		publishIPStateMetrics(state)
```

Comment:
inline, it is just as clear
```suggestion
		publishIPStateMetrics(service.buildIPState())
```

Reply:
Ack.